Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -70,7 +70,19 @@ class BuildCommandTestCases: CommandsBuildProviderTestCase {
                 // is what `binContents` is meant to represent.
                 return contents != ["output-file-map.json"]
             }
-            let moduleContents = (try? localFileSystem.getDirectoryContents(binPath.appending(component: "Modules"))) ?? []
+            var moduleContents: [String] = []
+            if buildSystemProvider == .native { 
```

Comment:
suggestion: instead of creating an `if` statement to control the behaviour of the `build` function, can we instead call a function that is overridden in the sub classes?